chart/genesis/values.yaml

```diff
@@ -0,0 +1,61 @@
+# Default values for genesis.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+secretMounts:
+  - name: 'docker-cert'
+    secretName: 'genesis-docker-cert'
+    mountPath: '/var/secrets'
+    readOnly: true
+
+replicaCount: 1
+
+image:
+  repository: gcr.io/whiteblock/genesis
```

Comment:
is this image correct?

Reply:
No, we no longer deploy to that endpoint

Reply:
ok my presumption was this was open-source code so deploying the public image would be ok.

I'll change it